src/Handler.php

```diff
@@ -176,7 +176,7 @@ public function installScenario($scenario, $dependencies, $dir)
         if ($status != 0) {
             return $status;
         }
-        passthru("composer -n --working-dir=$scenarioDir $scenarioCommand $extraOptions --prefer-dist --no-scripts", $status);
```

Comment:
Again, $scenarioCommand might be update. We should use --no-scripts with update.